test/e2e/fixtures/imported-account/state.json

```diff
@@ -443,6 +443,9 @@
         }
       }
     },
+    "TransactionController": {
```

Comment:
Nit: could this change be removed? I don't think it's needed anymore

Reply:
quite right.

Reply:
✅